build.gradle

```diff
@@ -434,14 +434,6 @@ configurations {
             force "com.github.luben:zstd-jni:${versions.zstd}"
             force "org.xerial.snappy:snappy-java:1.1.10.4"
             force "com.google.guava:guava:${guava_version}"
-
-            // TODO: Seems like this should be removable
-            force "org.apache.httpcomponents:httpclient-cache:4.5.13"
-            force "org.apache.httpcomponents:httpclient:4.5.13"
-            force "org.apache.httpcomponents:fluent-hc:4.5.13"
-            force "org.apache.httpcomponents:httpcore:4.4.16"
-            force "org.apache.httpcomponents:httpcore-nio:4.4.16"
-            force "org.apache.httpcomponents:httpasyncclient:4.1.5"
```

Comment:
Seems like there was conflict in CI for integration tests compilation:
```
Execution failed for task ':compileIntegrationTestJava'.
> Could not resolve all dependencies for configuration ':integrationTestCompileClasspath'.
   > Conflicts found for the following modules:
       - org.apache.httpcomponents:httpclient-cache between versions 4.5.14 and 4.5.13
       - org.apache.httpcomponents:httpclient between versions 4.5.14 and 4.5.13
       - org.apache.httpcomponents:httpcore between versions 4.4.16, 4.4.13 and 4.4.15
       - org.apache.httpcomponents:httpcore-nio between versions 4.4.16 and 4.4.15
```

Reply:
Looks like those resolutions are not optional, I'll follow up after the 2.11.0 release push